src/Yoti.Auth.Owin.nuspec

```diff
@@ -1,53 +1,39 @@
 <?xml version="1.0"?>
 <package >
   <metadata>
-    <id>Yoti.Auth.Owin</id>
-    <version>1.0.1</version>
-    <authors>Ben Harpin</authors>
-    <owners>Ben Harpin</owners>
+    <id>Yoti.Owin</id>
+    <version>2.0.0</version>
+    <authors>Yoti</authors>
+    <owners>Yoti</owners>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Yoti Owin .NET SDK</description>
-    <copyright>Copyright 2016</copyright>
+    <description>Yoti Owin .NET SDK. Yoti is an identity checking system that allows organisations to verify who people are.</description>
```

Comment:
Where did you get this text?

Reply:
https://www.yoti.com/developers/documentation